api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -17,7 +17,7 @@
 import java.util.Map;
 import java.util.Set;
 
-import org.apache.commons.lang.RandomStringUtils;
+import org.apache.commons.lang3.RandomStringUtils;
```

Comment:
Is this required by the ticket?

Reply:
no but its a dependency version that's suitable for apache velocity 2.1
http://velocity.apache.org/engine/2.1/dependencies.html

Reply:
if changing the UserServiceImpl is not required to do the update then please remove this change from the PR

Reply:
i made this change because` lang3.RandomStringUtils` is best suitable for `velocity 2.1` that we are upgrading to

Reply:
ok, but how does UserServiceImpl relate to velocity?

Reply:
@teleivo according to  http://commons.apache.org/proper/commons-lang/ it says `Note that Lang 3.0 (and subsequent versions) use a different package (org.apache.commons.lang3) than the previous versions (org.apache.commons.lang), allowing it to be used at the same time as an earlier version` and when i build minus the above change i get this [error](https://pastebin.com/4cpbDji7)